docs/source/posts/2023/29-05-2023-week-0-joaodellagli.rst

```diff
@@ -0,0 +1,31 @@
+The Beggining of Everything - Week 0
+===========================
```

Comment:
This `=` symbol  should cover the whole headline.

Something like:
```
The Begining of Everything - Week 0
===================================
```


![image](https://github.com/fury-gl/fury/assets/64432063/49e23e74-d144-4725-aa10-31dd79e9077b)